src/screens/VisualTests/SnapshotComparison.stories.tsx

```diff
@@ -147,6 +155,36 @@ export const SwitchingTests: Story = {
   },
 };
 
+export const InteractionFailure: Story = {
+  args: {
+    tests: [
+      makeTest({
+        status: TestStatus.Broken,
+        comparisons: [
+          {
+            id: "1",
+            browser: makeBrowserInfo(Browser.Chrome),
+            viewport: makeViewportInfo(1200),
+            result: ComparisonResult.CaptureError,
+            baseCapture,
+            headCapture: {
+              ...headCapture,
+              captureError: {
+                kind: "INTERACTION_FAILURE",
+                error: {
+                  name: "Error",
+                  message: `Unable to find an element by: [data-testid="button-toggle-snapshot"]`,
+                },
+              },
+            },
+            captureDiff,
+          },
+        ],
+      }),
```

Comment:
Can we just update the `makeTest` function to also allow the `BROKEN` status as input?